src/main.c

```diff
@@ -732,6 +759,15 @@ int main(int argc, char *argv[])
 	if (cfg.otp[0] != '\0')
 		log_debug("One-time password = \"%s\"\n", cfg.otp);
 
+	if (cfg.listen_port != 0){
+	    if(cfg.auth_id){
+	    	log_warn("auth-id will be ignored conflict with --ext-browser-saml");
+	    }
+		log_debug("Will listen on port \"%d\" for authentication id \n", cfg.listen_port);
+		free(cfg.auth_id);
+		cfg.auth_id = listen_for_id(&cfg);
+	}
+	
 	if (geteuid() != 0) {
 		log_error("This process was not spawned with root privileges, which are required.\n");
 		ret = EXIT_FAILURE;
```

Comment:
(Sorry, for putting it here, but I didn't find out how to comment on files that there not changed.)

This MR is missing changes to `README.md` and `doc/openfortivpn.1.in`